pandas/core/arrays/categorical.py

```diff
@@ -1579,7 +1579,7 @@ def argsort(self, ascending=True, kind="quicksort", *args, **kwargs):
         """
         return super().argsort(ascending=ascending, kind=kind, *args, **kwargs)
 
-    def sort_values(self, inplace=False, ascending=True, na_position="last"):
+    def sort_values(self, inplace=False, ascending=True, na_position="last", key=None):
```

Comment:
can you type the added keyword (bonus for typing other things)

Reply:
done.